documentation/crds_users_guide/source/library_use.rst

```diff
@@ -181,15 +181,15 @@ crds.get_default_context()
 ..........................
 
 `get_default_context()` returns the name of the pipeline mapping which is 
-currently the Latest Context
+currently the Build Context
 !!!!!!!!!!!!!!!!!!!
 
-*Latest Context* is the .pmap which is nominally in use by the pipeline.
-While it's common to make new files the latest as each context is added, it's
-possible for the *Latest Context* to lag behind the *Edit Context* when
-new files are being added but need additional testing in OPS. Deriving
-from the *Latest Context* is a crude kind of reversion since CRDS
-effectively branches around any existing subsequent contexts. use.
+*Build Context* is the .pmap which is nominally in use by the pipeline.
```

Comment:
Again, see above. I think it would best to add "build context" info, not replace latest with it. You could add to the first sentence: "*Latest Context* is the .pmap which is nominally in use by the Roman or HST pipeline. For JWST, see "build context" below.